### PR TITLE
Corrects actor name in README

### PR DESCRIPTION
**Pull request by sbussard**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -125,5 +125,3 @@ Of course, the recommended installation method is pip::
 Thanks for checking this library out! I hope you find it useful.
 
 Of course, there's always room for improvement. Feel free to `open an issue <https://github.com/kennethreitz/records/issues>`_ so we can make Records better, stronger, faster.
-
-
```
Comment (sbussard) [inline README.rst]: looks like atom editor removed these extra spaces

Comment (sbussard) [inline README.rst]: nbd if it's not to you

**Comment by jarek** [comment]
lol

**Comment by timofurrer** [comment]
It's not about the actor .. it's about cars .. as far as I understood.

**Comment by kennethreitz** [comment]
Yes, take a closer look :)

I may change it to "Aurthur Dent"

**Comment by sbussard** [comment]
HAHAHA ok ok I get it now

**Pull request closed without merging**